### ISSUE TITLE
Only some PCs are showing in dropdown

### ISSUE THREAD

**Issue by Aerandor** [Sepichat/FoundryVTT-GiveItem#10]
I didn't see any of the posted issues which addresses this problem, but I apologize if this was already addressed somewhere else that I missed.
When attempting to give an item, the dropdown list only shows two of the four player characters as options, even though all four were given the same configurations. I tried this with multiple different characters, but only the same two show up in the list. I am in GM mode while testing this. Any idea what would cause this? I'm using the 5e system and tidysheet5e module as well, if that helps.

**Comment by Sepichat** [comment]
Hello,
It might be related to #4 
Do all of the PC have a user assigned to already? The main criteria is for a PC to be associated to a player.
At first the module was displaying all the PCs (which could contain some unwanted "Characters"). It won't be related to tidy or the default sheet, but it should be only a "configuration issue" with the way it was designed following #4 
If you could provide a bit more information on your settings we can see how to handle this issue :)

**Comment by Aerandor** [comment]
Okay, so I played around with this a little and figured out a solution. You were correct about the connection to #4 in that the module was distinguishing between a "main" and an "owned" PC. I just wasn't making the mental connection in how to setup that designation internally. Simply setting a player as owner was not enough, it was also necessary for the player to select the token upon login under the player configuration, so that red lines demarcated the character before saving the configuration, like so:
 
![PlayerConfig](https://user-images.githubusercontent.com/28788106/120344031-718f8680-c2b6-11eb-866e-d2b003c2821e.png)

It seems that 2 of my players never realized they had to actually click on the character to select them, so I did this manually and now they show up in the dropdown.
